mercury_app/tests.py

```diff
@@ -136,6 +214,31 @@ def test_home(self):
         self.assertEqual(response.status_code, 302)
 
 
+class SelectEventsLoggedTest(TestBase):
+
+    def setUp(self):
+        super(SelectEventsLoggedTest, self).setUp()
+
+    @patch('mercury_app.views.get_api_organization', return_value=MOCK_ORGANIZATION_API.get('organizations'))
+    @patch('mercury_app.views.get_api_events_org', return_value=MOCK_EVENT_API.get('events'))
+    def test_screen_with_one_event(self, mock_get_api_events_org, mock_get_api_organization):
+        response = self.client.get('/select_events/')
+        self.assertEqual(response.status_code, 200)
+
+    @patch('mercury_app.views.get_api_events_id', return_value=MOCK_EVENT_API.get('events')[0])
+    def test_add_event(self, mock_get_api_events_id):
+        response = self.client.post('/select_events/', {'organization_id': '1234', 'organization_name': 'TestOrg'})
+        event = Event.objects.get(eb_event_id=MOCK_EVENT_API.get('events')[0].get('id'))
+        self.assertTrue(isinstance(event, Event))
+        self.assertEqual(response.status_code, 302)
+
+class SelectEventsRedirectTest(TestCase):
+
+    def test_redirect(self):
```

Comment:
If this is redirecting because you need session, please point it in the test name. 🙂